clients/unshaded/src/main/java/tachyon/client/WorkerBlockMasterClient.java

```diff
@@ -66,18 +66,18 @@ protected void afterConnect() {
    *
    * @param workerId the worker id committing the block
    * @param usedBytesOnTier the amount of used bytes on the tier the block is committing to
-   * @param tier the tier the block is being committed to
+   * @param tierAlias the alias of the tier the block is being committed to
```

Comment:
I am fine with calling them tierAlias,  as tier may refer to as the instance of `StorageTier`, while here the string value is in fact a "pointer"